###################error########################
やばい旨い

至福の時だった

スープがやばい（￣+ー￣）

お酒より飲んだ

づっと飲み続けれる本当にやばい

ここの常連と来たから何を注文したのかよく覚えてない

水炊きはMIXって言ってたようなー

茶碗蒸しも最高（￣+ー￣）

写真あり過ぎる

兎に角楽しい打ち上げにこんな旨いものあって幸せな時でしたーｖ（＾＿＾ｖ）♪

畑さんありがとう連れて来てくれて 
 Error: ;; Invalid input <+ + + 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
Traceback (most recent call last):
  File "/Users/itsp/Docker/DockerfileForWSD/workspace/NLPLibrary/nlplib_pyknp2.py", line 308, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/usr/local/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/usr/local/lib/python3.7/site-packages/pyknp/knp/blist.py", line 40, in __init__
    self.parse(spec)
  File "/usr/local/lib/python3.7/site-packages/pyknp/knp/blist.py", line 106, in parse
    raise Exception("Error: %s" % string)
Exception: Error: ;; Invalid input <+ + + 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
###########################################
###################error########################
*\/*


もう少し店員さんが笑顔だったらよいのにーと残念でした… 
 Error: ;; Invalid input <*\ *\ *\ 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
Traceback (most recent call last):
  File "/Users/itsp/Docker/DockerfileForWSD/workspace/NLPLibrary/nlplib_pyknp2.py", line 308, in pyknp_make_commentlist
    result = kparser.parse(sentence)
  File "/usr/local/lib/python3.7/site-packages/pyknp/knp/knp.py", line 83, in parse
    return BList(knp_lines, self.pattern, juman_format)
  File "/usr/local/lib/python3.7/site-packages/pyknp/knp/blist.py", line 40, in __init__
    self.parse(spec)
  File "/usr/local/lib/python3.7/site-packages/pyknp/knp/blist.py", line 106, in parse
    raise Exception("Error: %s" % string)
Exception: Error: ;; Invalid input <*\ *\ *\ 未定義語 15 その他 1 * 0 * 0 "品詞推定:名詞"
###########################################
